### PR TITLE
Child account management (FE)

### PR DESCRIPTION
**Pull request by EricBanker12**
# Description

As a Parent, I would like to have a place where I can manage my children's account and preferences, so that I can setup and customize the service for my children

[Trello Card](https://trello.com/c/gLrPV5Wd/21-parent-manage-my-childrens-account-and-preferences)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] Testing Rendering of Components

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by EricBanker12** [comment]
[[Parent] - Manage my children's account and preferences](https://trello.com/c/gLrPV5Wd/21-parent-manage-my-childrens-account-and-preferences)

File: src/app/components/dashboard/child-card.component.tsx [hunk 1]
```diff
@@ -1,19 +1,63 @@
 import React from 'react';
 import { Child } from '../../models';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
```
Comment (wSedlacek) [inline src/app/components/dashboard/child-card.component.tsx]: Where possible let's import like this:
```ts
import { Card, CardActions, ... } from '@material-ui/core'
```

**Review by ghost — COMMENTED** [review]
fixing of dashboard to match ui

**Review by ghost — COMMENTED** [review]
dashboard work up.

**Review by ghost — COMMENTED** [review]
rebasing of code.

**Review by EricBanker12 — COMMENTED** [review]
Everything seems to be working locally, and I found the code understandable. Blank tests are failing and will need to be made or removed

**Review by wSedlacek — APPROVED** [review]

**Review by EricBanker12 — COMMENTED** [review]
Manual testing appears that everything works fine, and all the tests pass. However, a lot of the tests have problems. Mainly, they're not awaiting and thus triggering stuff after the test. For the pages, we may need to mock out sub-components. And for components with api calls, we may need to mock the api call and await resolution/domChange.

We may also want to remove dashboard notification and left sidebar placeholders before presenting.

**Pull request merged**